Makefile

```diff
@@ -1,11 +0,0 @@
-all: delivery
```

Comment:
I'd prefer to keep the `Makefile` and to use it in `.travis.yml` as it make it easy to run the tests locally and be sure you are running what would run in travis.